packages/clerk-js/tsconfig.json

```diff
@@ -7,13 +7,12 @@
     "isolatedModules": true,
     "module": "esnext",
     "lib": ["es2021.intl"],
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
Only with that option the compiler was happy. Also, this applies: https://www.totaltypescript.com/tsconfig-cheat-sheet#not-transpiling-with-typescript